packages/sync/src/offline/OfflineRestoreHandler.ts

```diff
@@ -18,13 +17,13 @@ export const logger = debug.default(MUTATION_QUEUE_LOGGER);
 export class OfflineRestoreHandler {
 
   private apolloClient: ApolloClient<NormalizedCacheObject>;
-  private storage: PersistentStore<PersistedData>;
+  private storage: LocalForage;
   private readonly storageKey: string;
   private mutationCacheUpdates?: CacheUpdates;
 
   constructor(apolloClient: ApolloClient<NormalizedCacheObject>, clientConfig: DataSyncConfig) {
     this.apolloClient = apolloClient;
-    this.storage = clientConfig.storage as PersistentStore<PersistedData>;
+    this.storage = clientConfig.storage as LocalForage;
```

Comment:
Any reason for using LocalForage? Using this explicitly can affect user interface - they will be able to use LocalForage only when we use some methods that aren't in the PersistentStore interface.

Reply:
OK maybe I got it wrong but to me it looked like our PersistentStore interface only had 3 methods and all of these are covered properly by LocalForage. Was I wrong? Or do you mean what if they want to use something else as storage?